bindings_ffi/src/mls.rs

```diff
@@ -339,6 +339,14 @@ impl FfiConversations {
             }
         }
 
+        if let Some(ref image_url) = opts.group_image_url {
+            if !image_url.is_empty() {
+                convo
+                    .update_group_image_url(&self.inner_client, image_url.to_string())
```

Comment:
See comment in group name on creation PR - https://github.com/xmtp/libxmtp/pull/840#discussion_r1637095114